components/job/JobList.js

```diff
@@ -1,15 +1,15 @@
-import { useJobs } from 'store/jobs_store'
+import { useEffect, useState } from 'react'
 import { useProfileInfo } from 'store/profile_info'
 import { db } from 'utils/db'
 import getText from 'utils/i18n/Texts'
 
 const JobList = () => {
-  const jobs = useJobs((s) => s.jobs)
   const profileInfo = useProfileInfo((s) => s.profileInfo)
-
   const [activeApplications, setActiveApplications] = useState()
 
   useEffect(() => {
+    // TODO: What should happen with this fn? It's not being used anywhere.
+    // eslint-disable-next-line no-unused-vars
```

Comment:
cc @drewclem

Reply:
This component can be removed. It was original the dashboard view for company accounts, but that has been refactored